src/modules/commander/state_machine_helper.cpp

```diff
@@ -527,6 +537,10 @@ bool set_nav_state(vehicle_status_s &status, actuator_armed_s &armed, commander_
 				status.nav_state = vehicle_status_s::NAVIGATION_STATE_STAB;
 				break;
 
+			case commander_state_s::MAIN_STATE_AIRSPEED:
+				status.nav_state = vehicle_status_s::NAVIGATION_STATE_AIRSPEED;
```

Comment:
Should we only allow entering this mode if airspeed is currently valid?

Reply:
I think we should yes, it's of no use in case of invalid airspeed.
We can also think about using it as failsafe mode from Altitude mode in case of invalid altitude estimation.